### PR TITLE
Managed execution guide

### PR DESCRIPTION
**Pull request by discdiver**
Guide for managed execution. Intentionally unlinked during alpha.

navigate to guides/managed-execution/

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [X] This pull request includes tests or only affects documentation.
- [X] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [n/a] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed

### CODE REVIEW

File: docs/guides/managed-execution.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+description: Prefect will run your deployment on our infrastructure.
+tags:
+    - managed infrastructure
+search:
+  exclude: true
+---
+
+# Managed Execution
+
+Prefect Cloud provides a **prefect-managed** work pool that you can use for your workflow execution environment.
```
Comment (jakekaplan) [inline docs/guides/managed-execution.md:11]:
```suggestion
Prefect Cloud provides a **prefect:managed** work pool that you can use for your workflow execution environment.
```

File: docs/guides/managed-execution.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    Managed Execution is currently in alpha.
+    Features are likely to change without warning.
+
+## Usage guide
+
+Run a flow with managed infrastructure in three steps.
+
+### Step 1
+
+Create a new work pool of type **prefect-managed**.
```
Comment (jakekaplan) [inline docs/guides/managed-execution.md:27]:
```suggestion
Create a new work pool of type **prefect:managed**.
```

File: docs/guides/managed-execution.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+```python hl_lines="9" title="managed-execution.py"
+from prefect import flow
+
+if __name__ == "__main__":
+    flow.from_source(
+    source="https://github.com/desertaxle/demo.git",
+    entrypoint="flow.py:my_flow",
+    ).deploy(
+        name="test-managed-flow",
+        work_pool_name="prefect-managed",
```
Comment (jakekaplan) [inline docs/guides/managed-execution.md:47]: Users will be able to create work pools like normal, which also means they can't prefix it with `prefect`
```suggestion
        work_pool_name="my-managed-pool",
```

File: docs/guides/managed-execution.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+Schedule a deployment run from the UI or from the CLI.
+
+It may take a minute for the flow run to start and the logs to appear in the UI.
+
+That's it! You ran a flow on remote infrastructure without any infrastructure setup, worker, or cloud provider account.
+
+### Adding dependencies
+
+You can install Python package dependencies at runtime by passing `job_variables={"env": {"EXTRA_PIP_PACKAGES": ["pandas", "prefect-aws"] }}` like this:
```
Comment (jakekaplan) [inline docs/guides/managed-execution.md:72]: Is it worth mentioning the pull_step to do this or should we not even get into it? Theres no benefit that I know of from doing that vs this, except you already have your requirements file

File: docs/guides/managed-execution.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+## Limitations
+
+Managed execution requires Prefect 2.14.4 or newer.
+
+All limitations listed below may change without warning during the alpha period.
+We will update this page as we make changes.
+
+### Concurrency & work pools
+
+- Maximum of 10 concurrent flow runs per workspace.
```
Comment (jakekaplan) [inline docs/guides/managed-execution.md:97]:
```suggestion
- Maximum of 10 concurrent flow runs per workspace across all `prefect:managed` pools
```

File: docs/guides/managed-execution.md (hunk 4 above)
Comment (discdiver) [reply docs/guides/managed-execution.md:72]: I'll add that it's an alternative but not show it.

**Review by jakekaplan — APPROVED** [review]

**Review by urimandujano — APPROVED** [review]
As someone with low prior understanding of the managed execution offering, this doc was clear for me!

**Pull request merged**